src/main/java/core/basesyntax/Buldozer.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public class Buldozer extends Machine{
+    @Override
+    public void doWork() {
+        System.out.println("Buldozer is working!");
+    }
+    @Override

```

Comment:
```suggestion
    }
    
    @Override
```
між методами краще ставити пусту строку для читаймості